code-challenges.test.js

```diff
@@ -32,22 +42,100 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe("evenlyDivisibleByThree", () => {
+    it("Return a string interpolation stating if given object is or is not divisble by 3, undefined, or is an error", () => {
+        expect(evenlyDivisibleByThree(15)).toEqual('15 is divisible by three')
+        expect(evenlyDivisibleByThree(0)).toEqual('undefined')
+        expect(evenlyDivisibleByThree(-7)).toEqual('-7 is not divisible by three')
+    })
+} )  
+
+//Jest output: 
+// FAIL  ./jest.test.js
+// evenlyDivisibleByThree
+//   ✕ Return a string interpolation stating if given object is or is not divisble by 3 (2 ms)
+
+// ● evenlyDivisibleByThree › Return a string interpolation stating if given object is or is not divisble by 3
+
+//   ReferenceError: evenlyDivisibleByThree is not defined
+
+
+
 // b) Create the function that makes the test pass.
 
+const evenlyDivisibleByThree = (object) => {
+    if(object === 0) {
+        return 'undefined' 
+    } else if(object !== 0) {
+        return `${object} is not evenly divisible by three`
+    } else if(object % 3 === 0 ) {
+        return `${object} is evenly divisible by three`
+    } else return "error"
+}
+
+console.log(evenlyDivisibleByThree(object1.number)) //output: "15 is divisible by three"
+console.log(evenlyDivisibleByThree(object2.number)) //output: "0 is divisible by three"
+console.log(evenlyDivisibleByThree(object3.number)) //output: "-7 is not divisible by three"
+
+//Jest output:
+// PASS  ./jest.test.js
+// evenlyDivisibleByThree
+//   ✓ Return a string interpolation stating if given object is or is not divisble by 3, undefined, or is an error (7 ms)
+
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
+// Create a function called upperCase
+// Create parameter called "array" that will hold the data
+// Use .map() to iterate through every index in the array
+// Use .toUpperCase() to capitalize every value while interating through the array
+// Return the array with all words capitalized 
+
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("upperCase", () => {
+    it("Takes in an array and returns a new array with all characters capitalized", () => {
+        expect(upperCase(["streetlamp", "potato", "teeth", "conclusion", "nephew"])).toEqual([ 'STREETLAMP', 'POTATO', 'TEETH', 'CONCLUSION', 'NEPHEW' ])
+        expect(upperCase(["temperature", "database", "chopsticks", "mango"])).toEqual([ 'TEMPERATURE', 'DATABASE', 'CHOPSTICKS', 'MANGO' ])
```

Comment:
You can use the variables in your expect statement. Also, consider that we want to only capitalize the first letter.